chcemvediet/apps/anonymization/utils.py

```diff
@@ -1,12 +1,142 @@
+import re
 import tempfile
 import shutil
 from contextlib import contextmanager
+from xml.etree import ElementTree
 
 
+ANONYMIZATION_STRING = r'xxxxx'
+
+TRANSLATE_TABLE = {
+    # Slovak diacritics
```

Comment:
@martinmacko47 Tabulka ktora sluzi pre slovensku diakritiku ktora prepoji pismenka napr .[aAáÁ]. Uvedené poznámky sú dosť dôležité, ale bolo by ich asi lepšie prepísať niekde do dokumentácie.